pantry-finder-api/app/models/zip_agency.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# zip codes the agency services through events
+class ZipAgency < ApplicationRecord
```

Comment:
I would expect this class to be named something like `EventZipCode`. The fact that it's related to an agency is an emergent property that you get because of the existing relationship with events. The table is actually the join between Events and ZipCodes.

Reply:
Makes sense. From a pure data architecture standpoint the event_service_geographies table can support multiple types of geographies (Zip code, County, etc...) that is assigned to an event.

Reply:
EventZipCode it is